tutorials/dlp-hybrid-inspect/src/main/java/com/example/dlp/Database.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2021 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.dlp;
+
+public class Database {
```

Comment:
Consider using Autovalue for this: https://github.com/google/auto/blob/master/value/userguide/index.md

Reply:
Sounds good. I'll try moving to this.

Reply:
Ran into an issue with AutoValue and Gson so instead moved this into a static inner class